README.md

```diff
@@ -406,28 +406,46 @@ you should use any version `<5.5 >=5.7.1` as the versions in-between have some n
 
 The NPM packager supports the following `packagerOptions`:
 
-| Option             | Type   | Default               | Description                                         |
-| ------------------ | ------ | --------------------- | --------------------------------------------------- |
-| ignoreScripts      | bool   | false                 | Do not execute package.json hook scripts on install |
-| noInstall          | bool   | false                 | Do not run `npm install` (assume install completed) |
-| lockFile           | string | ./package-lock.json   | Relative path to lock file to use                   |
+| Option                  | Type     | Default             | Description                                                         |
+|-------------------------|----------|---------------------|---------------------------------------------------------------------|
+| ignoreScripts           | bool     | false               | Do not execute package.json hook scripts on install                 |
+| noInstall               | bool     | false               | Do not run `npm install` (assume install completed)                 |
+| lockFile                | string   | ./package-lock.json | Relative path to lock file to use                                   |
+| copyPackageSectionNames | string[] | []                  | Entries in your `package.json` to copy to the output `package.json` |
```

Comment:
Just by reading that line, it's not clear how it'll be used. Maybe you can put sth as example? Like
```suggestion
| copyPackageSectionNames | string[] | []                  | Entries in your `package.json` to copy to the output `package.json` (ie: ESM output) |
```

Reply:
👍 good point. That should help tie this to the section below explaining ESM output too